Enterprise-CMCS/macpro-mdct-core
# Disable Dependabot dependency mangement in core

### Description
To deprecate dependabot manually I did the following 
1) settings at the top of the repo in GitHub
2) code and security analysis on the left hand side 
3) Click disable on dependabot 
![Screenshot 2024-03-22 at 3 54 58 PM](https://github.com/Enterprise-CMCS/macpro-mdct-core/assets/52459927/3160d056-4cb3-4485-923c-21e33f2a55bb)


In addition I removed the dependabot workflow from the .github/workflows folder 
Also, I removed an OBE line in the snyk auto merge workflow and renamed the step 



### Related ticket(s)
https://jiraent.cms.gov/browse/CMDCT-3083

---
### How to test
Go to the security tab at the top of the repo click dependabot on the left side and verify that issues are no longer being reported.
ensure snyk PR's still come through


### Important updates
none


---
### Author checklist


---
